adversarial_workflow/utils/citations.py

```
@@ -0,0 +1,608 @@
+    if output_path:
+        output_path.parent.mkdir(parents=True, exist_ok=True)
+        with open(output_path, "w") as f:
+            f.write(content)
+
+    return content
+
+
+def verify_document(
+    document_path: Path,
+    output_tasks_path: Optional[Path] = None,
+    mark_inline: bool = True,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)